packages/data-point-express/README.md

```diff
@@ -102,8 +102,8 @@ const app = new express()
 Service.create({
   // add DataPoint entities
   entities: {
-    'entry:HelloWorld': (acc) => 'Hello World!!',
-    'entry:Greet': (acc) => `Hello ${acc.locals.params.person}!!`
+    'entry:HelloWorld': (value, acc) => 'Hello World!!',
```

Comment:
there are many instances where you will see this (value, acc) or even (value, acc, next) I did a search replace, there were too many specific cases where sometimes acc was used other than acc.value, and the codemods cant handle many of our use cases on tests, so decided to just refactor all to be (value, acc, [next])